compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Pretty.hs

```diff
@@ -257,17 +257,17 @@ instance Pretty BuiltinExpr where
     BEGreaterEqNumeric -> "GEQ_NUMERIC"
     BEGreaterNumeric -> "GREATER_NUMERIC"
     BENumericFromText -> "FROM_TEXT_NUMERIC"
```

Comment:
Would it be consistent to call this `NUMERIC_FROM_TEXT` or maybe `TEXT_TO_NUMERIC`?

Reply:
In another PR.

Reply:
:+1: